### PR TITLE
Fixes #10: Clean-the-current project-to-show-Hello-world

### LINKED ISSUE #10: Clean the current project to show Hello world
Clean the current project to show Hello world

### PR DESCRIPTION
**Pull request by KvngJamesNG**
1. Update index.html
2. Remove App.css
3. Update App.jsx to show Hello world
4. Update index.css
5. Remove react.svg 
6. Remove vite.svg

Screen Shot 2024-08-20 at 10 58 02 AM

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (rt4914) [inline index.css:3]: Add blank line at the end.

File: App.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react'
```
Comment (rt4914) [inline App.jsx:1]: Remove this import. It is not getting used.

File: App.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+function App() {
```
Comment (rt4914) [inline App.jsx:3]: Also, convert this into an arrow function, we will be using arrow function across the app.

File: index.html [hunk 4]
```diff
@@ -1,10 +1,7 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
```
Comment (rt4914) [inline index.html:4]: Don't remove this.

File: index.html [hunk 5]
```diff
@@ -1,10 +1,7 @@
...
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (rt4914) [inline index.html:6]: Keep this `meta` tag too.

**Review by rt4914 — COMMENTED** [review]
All the files that you have created already exists in the project. Please have a look at src folder and make changes to already existing files.

File: index.css (hunk 1 above)
Comment (KvngJamesNG) [reply index.css:3]: done.. Kindly review PR

File: App.jsx (hunk 2 above)
Comment (KvngJamesNG) [reply App.jsx:1]: done... Kindly review PR

File: App.jsx (hunk 3 above)
Comment (KvngJamesNG) [reply App.jsx:3]: done... Kindly review PR

File: index.html (hunk 4 above)
Comment (KvngJamesNG) [reply index.html:4]: done... Kindly review PR

File: index.html (hunk 5 above)
Comment (KvngJamesNG) [reply index.html:6]: done... Kindly review PR

**Pull request closed without merging**